syncmymoodle/__main__.py

```diff
@@ -18,6 +18,7 @@
 import getpass
 import pdfkit
 import shutil
+from pathlib import Path
```

Comment:
for consistency we should probably use pathlib instead of the os.path.join commands.

Reply:
Sure. The config part was done rather quick and I did not want to touch more parts than required but this is definitely on the radar.